pennylane/ops/op_math/sum.py

```diff
@@ -128,9 +131,15 @@ def __init__(self, *summands, do_queue=True, id=None):  # pylint: disable=super-
         if len(summands) < 2:
             raise ValueError(f"Require at least two operators to sum; got {len(summands)}")
 
-        self.summands = summands
-        self.data = [s.parameters for s in summands]
-        self._wires = qml.wires.Wires.all_wires([s.wires for s in summands])
+        self.summands: List[Operator] = []
+        for summand in summands:
+            if isinstance(summand, Sum):
+                self.summands.extend(summand.summands)
+                continue
+            self.summands.append(summand)
+
+        self.data = [s.parameters for s in self.summands]
+        self._wires = qml.wires.Wires.all_wires([s.wires for s in self.summands])
```

Comment:
I think for now we can leave the simplification logic out. This should be addressed by a separate PR, and we still need to figure out if we want to simplify during each construction automatically, or if its better to provide a top level function to simplify in general 👍🏼 

```suggestion
        self.summands = summands
        self.data = [s.parameters for s in self.summands]
        self._wires = qml.wires.Wires.all_wires([s.wires for s in self.summands])
```